### DISCUSSION TITLE
Problem with audio stream / icecast2

### DISCUSSION THREAD

**Discussion by jmtmp** [Nachtzuster/BirdNET-Pi#211] [Q&A]
I have been using Birdnet for several months but today I made new frresh instalation of BirdNet on Raspi4 with Bookworm 64 lite.

But there is issue with audio/livestream and I saw following rows in installation log: 

   Setting up icecast2 (2.4.4-4+b1) ...
   update-rc.d: error: no runlevel symlinks to modify, aborting!

   ++ LogLevel_BirdnetRecordingService=error
   ++ LogLevel_LiveAudioStreamService=error
   ++ LogLevel_SpectrogramViewerService=error

Do you know how to fix it?

**Comment by lloydbayley** [comment]
~It's possible it's a botched install.. Not you per se but when it was collecting and installing things. Maybe a resource was out of reach... Can you try installing again?~

**Comment by lloydbayley** [comment]
Actually, I don't know. I'm doubting myself now. 
Anyway, there should be someone else better qualified to answer you out there.
Good luck.
